ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessRewriter.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.restmodel.displayedgraph
+
+import pl.touk.nussknacker.engine.graph.node.{Disableable, NodeData}
+import pl.touk.nussknacker.restmodel.displayedgraph.rewriter.{RemoveAndBypassNode, RemoveBranch}
+
+object ProcessRewriter {
+
+  def removeAndBypassNode(process: DisplayableProcess, node: NodeData): DisplayableProcess =
```

Comment:
Why `removeAndBypassNode` and `removeBranch` are here? They are used someway? `RemoveAndBypassNode` and `RemoveBranch` are rather implementation details so IMHO they should be package private protected and here should be only one method.